Source/WebCore/platform/graphics/cg/ImageBufferIOSurfaceBackend.h

```diff
@@ -79,9 +79,9 @@ class WEBCORE_EXPORT ImageBufferIOSurfaceBackend : public ImageBufferCGBackend {
 
     unsigned bytesPerRow() const override;
 
-    void finalizeDrawIntoContext(GraphicsContext& destinationContext) override;
```

Comment:
Is this the only impl of this function that's been removed? Can we drop the callsite and decl from ImageBufferBackend.h?

Reply:
I left it due to it's not entirely clear yet if we do not need it in the very near future.
We can remove it in a separate patch. I'll do it if nobody else will after near future.